core-plugins/src/main/java/co/cask/hydrator/plugin/transform/LogParserTransform.java

```diff
@@ -119,60 +120,69 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws Ille
 
   @Override
   public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter) throws Exception {
-    String log = getLog(input);
-    if (log == null) {
-      LOG.debug("Couldn't read schema, log message was null");
-      return;
-    }
-
-    StructuredRecord output;
-    if (S3_LOG.equals(config.logFormat)) {
-      Matcher logMatcher = S3_LOG_PATTERN.matcher(log);
-      if (!logMatcher.matches() || logMatcher.groupCount() < S3_REGEX_LENGTH) {
-        LOG.debug("Couldn't parse log because log did not match the S3 format, log: {}", log);
-        return;
-      }
-      output = parseRequest(logMatcher, S3_INDICES);
-    } else if (CLF_LOG.equals(config.logFormat)) {
-      Matcher logMatcher = CLF_LOG_PATTERN.matcher(log);
-      if (!logMatcher.matches() || logMatcher.groupCount() < CLF_REGEX_LENGTH) {
-        LOG.debug("Couldn't parse log because the log did not match the CLF format. log: {}", log);
-        return;
-      }
-      output = parseRequest(logMatcher, CLF_INDICES);
-    } else {
-      if (log.startsWith("#")) {
-        LOG.trace("Log is a comment. Ignoring...");
+    try {
```

Comment:
In this case, let's not wrap all transform methods in a try catch. The reason being that if something really bad happens in the Transform function that you weren't explicitly emitting errors for, the pipeline should still fail so you can update your code. So remove the try catch wrapper for the transform function.